src/rpc/server.cpp

```diff
@@ -492,13 +492,19 @@ std::vector<std::string> CRPCTable::listCommands() const
     return commandList;
 }
 
-UniValue CRPCTable::dumpArgMap() const
+UniValue CRPCTable::dumpArgMap(const JSONRPCRequest& args_request) const
 {
+    JSONRPCRequest request(args_request);
+    request.mode = JSONRPCRequest::GET_ARGS;
+    request.params = UniValue();
```

Comment:
question: Any reason to reset those?

Reply:
re: https://github.com/bitcoin/bitcoin/pull/21035#discussion_r580903807

> question: Any reason to reset those?

Not really, removed this. This was copied from `CRPCTable::help` but that method sets `method` and `params` fields together, and here it doesn't make sense to set just the one field.